### ISSUE TITLE
Crashes on client

### ISSUE THREAD

**Issue by Niterux** [sakura-ryoko/afkplus#11]
I wanted to use a world sharing (e4mc, world host, ESSENTIAL, etc) mod to play with my friend and add this afk mod so they could know when I'm afk but when I put this in my mods the game crashes
https://mclo.gs/zM5BFGp

**Comment by sakura-ryoko** [comment]
Yeah I originally tested using the "main" space for the Client functionality, which seems to work fine using the vanilla launcher.  I'll look at this later today.

**Comment by sakura-ryoko** [comment]
please test version 1.3.6 from github releases .

**Comment by Niterux** [comment]
Yes it is working now on the client with this new release, thank you very much ❤️
